src/__tests__/element-queries.js

```diff
@@ -565,4 +565,19 @@ test('getByText ignores script tags by default', () => {
   expect(getAllByText(/hello/i, {ignore: false})).toHaveLength(3)
 })
 
+test('get element by its dynamically assigned value', () => {
```

Comment:
Could we also have a test for a textarea and a select. Can be in this test or in a new one I don't care which.